odoo/addons/base/models/ir_qweb_fields.py

```diff
@@ -142,13 +142,14 @@ def get_available_options(self):
         options = super(IntegerConverter, self).get_available_options()
         options.update(
             format_decimalized_number=dict(type='boolean', string=_('Decimalized number')),
+            decimal=dict(type='integer', string=_('Decimal Points')),
```

Comment:
Maybe we could name the option "precision_digits" to make it a bit more clear in template usage?
(To match the naming we use for "float_compare").